spec/testsSpec.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+// Write tests below:
+
+// e.g.
+describeClass('Arithmetic1', function() {
+  describe('Sum', function(){
+    it('3 + 4 equals 6', function() { 
+      expect(3+4).toEqual(6);
+    });
+  });
+});
+
```

Comment:
Perhaps could have written more tests for functions? However, not sure if that's needed.